src/three/math/Ellipsoid.d.ts

```diff
@@ -1,9 +1,5 @@
 import { Vector3, Matrix4 } from 'three';
 
-export const WGS84_RADIUS;
-export const WGS84_FLATTENING;
-export const WGS84_HEIGHT;
-
```

Comment:
Thanks for the catch! Can we add these to a constants.d.ts file and export that from index.d.ts, as well, then?

Reply:
Thanks!